src/DetailsView/components/Issues-details-pane.tsx

```diff
@@ -68,14 +71,29 @@ export class IssuesDetailsPane extends React.Component<IssuesDetailsPaneProps, I
     }
 
     private getFileIssueDetailsButton(issueData: CreateIssueDetailsTextData): JSX.Element {
-        return (
+        const oldButton: JSX.Element = (
```

Comment:
not thrill with the names but I don't have any better suggestion. Maybe `oldExperienceButton` and `newExperienceButton`?